### PR TITLE
Correct some typos and rewrite confusing setences

### PR DESCRIPTION
**Pull request by lcsataides**
Hien! So, it's mostly the first iterate I made, most of it are basically typos so I'd rather look for the "too long" sentences that I rewrote it all.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,33 +73,31 @@ We strongly recommend reading the full [RSCSS](http://rscss.io) documentation to
...
-[4 Simple Steps to Vertical Rhythm](http://typecast.com/blog/4-simple-steps-to-vertical-rhythm)
-[Aesthetic Sass 3: Typography and Vertical Rhythm](https://scotch.io/tutorials/aesthetic-sass-3-typography-and-vertical-rhythm)
-[CSS Baseline: The Good, The Bad And The Ugly](https://www.smashingmagazine.com/2012/12/css-baseline-the-good-the-bad-and-the-ugly/)
-[Using Sass & Compass Vertical Rhythm to set up typography defaults in a project.](https://medium.com/@amlinarev/using-sass-compass-vertical-rhythm-to-set-up-typography-defaults-in-a-project-34fe2f1d2c02#.4qrws1cww)
+**Furhter reading on this subject** [4 Simple Steps to Vertical Rhythm](http://typecast.com/blog/4-simple-steps-to-vertical-rhythm) [Aesthetic Sass 3: Typography and Vertical Rhythm](https://scotch.io/tutorials/aesthetic-sass-3-typography-and-vertical-rhythm) [CSS Baseline: The Good, The Bad And The Ugly](https://www.smashingmagazine.com/2012/12/css-baseline-the-good-the-bad-and-the-ugly/) [Using Sass & Compass Vertical Rhythm to set up typography defaults in a project.](https://medium.com/@amlinarev/using-sass-compass-vertical-rhythm-to-set-up-typography-defaults-in-a-project-34fe2f1d2c02#.4qrws1cww)
```
Comment (lcsataides) [inline README.md]: Just found it: Further* somehow it didn't entered :(

File: README.md [hunk 2]
```diff
@@ -412,27 +434,32 @@ color: fuchsia;
...
+- **unclassed.sass**: set styles for baisc html tags, those who don't usually have a class.
+- **mixins.sass**: contains structures that repeats throughout the project classes.
+
+### 3\. Components
+
+Contains two different groups of stylesheets (**elements** and **modules**) which are the styles specific to the project architecture/structure. **Basic of files:**
+
 - exemplos aqui pfv, obgd dnd
 
+  > pretty confusing for me, but i'd rather make this first iterate better and only then get back here, k?*
```
Comment (lcsataides) [inline README.md]: subject mapping for next 4dx/iterate

File: README.md [hunk 3]
```diff
@@ -273,9 +287,14 @@ $font-size-tiny: .8em;
 $font-size-minuscule: 0.6em;
 ```
 
-Attribute font sizes to your html tag selectors so you will have to worry about attributing font sizes on very specific classes or situations.
+~~Attribute font sizes to your html tag selectors so you won't have to worry about attributing font sizes on very specific classes or situations.~~
```
Comment (lcsataides) [inline README.md]: Can anyone support me on this changes? I actually rewrite it and I'm not so sure of what was intended before. @caepenna @carlosshaka @miprotzek @wkurosawa

Comment (caepenna) [inline README.md]: Hm, indeed it is a bit confusing, but I get the idea. I suggest rephrasing it the following way:

"Use the font size attribute on html tag selectors (`h1`, `h2`, `p`, etc) so you won't have to do so on specific situations unless it is an exception."

**Pull request merged**